newIDE/app/src/EventsSheet/EnumerateEventsMetadata.js

```diff
@@ -0,0 +1,28 @@
+import flatten from 'lodash/flatten';
+import { mapFor } from '../Utils/MapFor';
+const gd = global.gd;
+
+export const enumerateEventsMetadata = () => {
```

Comment:
Last thing to refactor: 

Add //@flow at the top of this file, then export the type:

```
export type EventMetadata = {|
    type: string,
    fullName: string,
    description: string,
|};
```

we'll then use it in the EventsSheet :)

Reply:
You can also specify that enumerateEventsMetadata returns an array of EventMetadata:

`export const enumerateEventsMetadata = (): Array<EventMetadata> => {... `

This will help to ensure that we're properly using the array in EventsSheet.

(if someone try to do something wrong with the array or its content, flow will warn us).